src/handlers/swap_handlers.ts

```diff
@@ -449,6 +450,10 @@ const parseSwapQuoteRequestParams = (req: express.Request, endpoint: 'price' | '
             },
         ]);
     }
+    const priceImpactProtectionPercentage =
+        req.query.priceImpactProtectionPercentage === undefined
+            ? DEFAULT_PRICE_IMPACT_PROTECTION_PERCENTAGE
+            : Number.parseFloat(req.query.priceImpactProtectionPercentage as string);
```

Comment:
do we expect percentage or decimal from the api user? 

for 20%, 20? or 0.2?

Reply:
we are expected `0.2` from the user for 20%

Reply:
I just realized that `slippagePercentage` also uses decimal 😞. I guess we are sort of stuck with this convention.

I think we should add a validation on the range if we already don't have one..
If a user wanted 20% and provide 20 by mistake then that could disable price impact protection altogether.